libs/nx-docusaurus/README.md

```diff
@@ -1,7 +1,71 @@
-# nx-docusaurus
+<p align="center"><img src="https://raw.githubusercontent.com/nrwl/nx/master/nx-logo.png" height="190">
+<img src="https://raw.githubusercontent.com/encharm/Font-Awesome-SVG-PNG/master/black/png/64/plus.png" height="150">
+<img src="https://raw.githubusercontent.com/facebook/docusaurus/master/website/static/img/docusaurus.png" height="200"></p>
 
-This library was generated with [Nx](https://nx.dev).
+<hr>
 
-## Running unit tests
+# What is Nx-Docusaurus?
 
-Run `ng test nx-docusaurus` to execute the unit tests via [Jest](https://jestjs.io).
+It's in the name! Nx-Docusaurus is a package that aims to provide first-class support for Docusaurus in an Nx powered monorepo.
+
+# Getting Started
+
+## Add the Plugin
+
+```
+# With npm
+npm install nx-docusaurus
+
+# With yarn
+yarn add nx-docusaurus
+```
+
+## Generate a Docusaurus application
+
+```
+nx g nx-docusaurus:nxDocusaurus my-docs-app
+```
+
+## Serve the application
+
+```
+nx run my-docs-app:docusaurus
+```
+
+## Build the application
+```
+nx run my-docs-app:build-docusaurus
+```
+
+
+# Usage
+
+## Docusaurus schematic
+
+`nx g nx-docusaurus:nxDocusaurus <name> [...options]`
+
+| Options           | Default | Description                                |
+| ----------------- | ------- | ------------------------------------------ |
+| `--tags`          | -       | Tags to use for linting (comma-delimited). |
+| `--directory`     | 'apps'  | A directory where the project is placed.   |
+
+## Docusaurus server builder
+
+`nx run my-docs-app:docusaurus [..options]`
```

Comment:
You're missing a third dot in front of `options`. Same for the following command.